AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/src/HierarchicalVersionUtils.dfy

```diff
@@ -111,4 +126,59 @@ module HierarchicalVersionUtils {
     assume {:axiom} fresh(Crypto) && fresh(Crypto.Modifies);
     return Success(Crypto);
   }
+
+  method ValidateMdDigest (
+    plainText: KMS.PlaintextType,
```

Comment:
Suggestion: There are so many plain-texts in our lives;
plain-text messages/content, 
plain-text data keys,
plain-text wrapping keys...
let's give this variable a little distinctiveness by calling it plainTextTuple;
we know it is the concatenation of the PDK + protected-MD-Digest;
we could call that concatenation a tuple for short (
it sort of is, sort of isn't, but I think tuple is good enough and better than plainText).
```suggestion
    plainTextTuple: KMS.PlaintextType,
```